tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AzureFunctionsTests.cs

```diff
@@ -177,7 +177,9 @@ public InProcessRuntimeV4(ITestOutputHelper output)
         [Trait("RunOnWindows", "True")]
         public async Task SubmitsTraces()
         {
-            using var agent = EnvironmentHelper.GetMockAgent(useTelemetry: true);
+            using var agent = EnvironmentHelper.GetMockAgent(useTelemetry: true, useStatsD: true);
+            // in this case and version, azure registers an event listener, as runtime metrics does, we can't have 2 listeners, it will crash
```

Comment:
Does this only crash in tests, or should we forcibly disable runtime metrics when the tracer is running in Azure Functions? (not in this PR, just checking)

Reply:
It technically works in Azure but if Azure registers its logforwarder event listener, which seems to happen in the InProcess test in .net 6, it will provoke a fatal clr crash when application exits, on shutdown.. but in this PR, we don't activate runtime metrics yet, the idea for later is to instrument the EventListener.Dispose method up to net8.0 to bail out before it does anything...

Reply:
Thanks for clarifying.